### PR TITLE
Merge maint-1.0 to maint-1.1

### PR DESCRIPTION
**Pull request by rljacob**
Merge of maint-1.0 in to maint-1.1
    
Bring several changes made to maint-1.0 to the maint-1.1 branch

Major changes:
Convert all use cases to use netcdf3 input files.
Add SCORPIO external and use it.
Add some CMPI6 use cases.

Summary:

EAM:
    Add use cases (and compsets):
       2010_cam5_CMIP6-HR.xml
       2010_cam5_CMIP6-LR.xml
       1850S_cam5_CMIP6.xml
    Update CMIP6_bgc use cases and others to use netcdf3
    Modify cam_diagnostics to output at more pressure levels.
    Adds AODALL to compute total aerosol (incl. nighttime) optical depth
    
CICE:
       Fix CICE restart fail - freeing iodesc
ELM:
       Land init fix for missing mask
Driver:
       mpi tag fix needed for ensembles in new tests.
I/O:  (most of the files touched are from adding SCORPIO)
       Add SCORPIO external directories and enable PIO2 as option in all models and
        make box rearranger the default.
    
CIME scripts:  Fix bug in cime macrowriterbase, Fix implementation of SVN server,
   Sync wget from master, Ignore unicode chars from command output, update check_input_data
    to prevent server outage from crashing case.submit
    
testing:   add transient test to production, allow netcdf3 in v1cmip testmod,
    Big update to wait_for_tests/jenkins, Adds tsc and npg tests
    
config_batch:   Update queue policy on theta
config_inputdata:  Remove old ornl svn data server
config_compilers:   remove cab,  update theta:intel, update compy:intel, add compy:pgi add PIO2 flags.
config_machines:   update compy to be same as master

### CODE REVIEW

**Comment by rljacob** [comment]
This has not been tested yet but wanted to get some eyeballs on it.   I made a branch off of maint-1.1, did the merge to that and resolved conflicts.

File: cime/scripts/lib/CIME/case/check_input_data.py [hunk 1]
```diff
@@ -2,7 +2,11 @@
 API for checking input for testcase
 """
 from CIME.XML.standard_module_setup import *
+<<<<<<< HEAD
```
Comment (jgfouca) [inline cime/scripts/lib/CIME/case/check_input_data.py]: This is definitely not right. Looks like conflict metadata was committed.

Comment (rljacob) [reply cime/scripts/lib/CIME/case/check_input_data.py]: Ok that should be fixed now.

**Comment by rljacob** [comment]
I having trouble building any case on this branch.  Probably something in the merge of CIME lib files.
maint-1.0 had  CIME 5.4.0-alpha.27 plus some modifications by @jgfouca .   maint-1.1 had cime5.6.0 and no mods that I can tell.  

The error message is "ERROR: Multiple matches"

**Comment by jonbob** [comment]
@rljacob - I have seen that before and have a vague memory of what caused it. I'll look and see if I can find it

**Comment by singhbalwinder** [comment]
I am not sure which machine are you using....is the "Multiple matches" error for queues on the system? If it is, we can revisit the queues in config_batch.xml for that particular machine.

**Comment by jonbob** [comment]
When we saw that error before, it was due to a syntax problem in CAM's config_compset file?

**Comment by rljacob** [comment]
I'm doing everything on compy.  I've seen this when config_pesall.xml can't define a clear pelayout but I reverted that file back to the maint-1.1 head.

**Comment by jayeshkrishna** [comment]
Looks like duplicate tags in the xml files during merge caused the issue, with the quick hack below I was able to get create_newcase working on compy

```
diff --git a/cime/config/e3sm/machines/config_batch.xml b/cime/config/e3sm/machines/config_batch.xml
index 18422af..b259db0 100644
--- a/cime/config/e3sm/machines/config_batch.xml
+++ b/cime/config/e3sm/machines/config_batch.xml
@@ -389,12 +389,6 @@
     </queues>
    </batch_system>
 
-   <batch_system MACH="compy" type="slurm">
-    <queues>
-      <queue walltimemax="00:59:00" default="true">slurm</queue>
-    </queues>
-   </batch_system>
-
    <batch_system MACH="sooty" type="slurm" >
      <directives>
        <directive>--ntasks-per-node={{ tasks_per_node }}</directive>
diff --git a/cime/config/e3sm/machines/config_machines.xml b/cime/config/e3sm/machines/config_machines.xml
index 20ecc8c..1c641a7 100644
--- a/cime/config/e3sm/machines/config_machines.xml
+++ b/cime/config/e3sm/machines/config_machines.xml
@@ -1955,8 +1955,8 @@
         <command name="load">mkl/2019u5</command>
       </modules>
     </module_system>
-    <RUNDIR>$CIME_OUTPUT_ROOT/$CASE/run</RUNDIR>
-    <EXEROOT>$CIME_OUTPUT_ROOT/$CASE/bld</EXEROOT>
+    <!--RUNDIR>$CIME_OUTPUT_ROOT/$CASE/run</RUNDIR-->
+    <!--EXEROOT>$CIME_OUTPUT_ROOT/$CASE/bld</EXEROOT-->
     <TEST_TPUT_TOLERANCE>0.05</TEST_TPUT_TOLERANCE>
     <MAX_GB_OLD_TEST_DATA>1000</MAX_GB_OLD_TEST_DATA>
     <environment_variables>

```

**Comment by jayeshkrishna** [comment]
There are also some extra ">" in some of the compy xml tags,

```
-      <modules mpilib="impi" compiler="intel">>
+      <modules mpilib="impi" compiler="intel">
         <command name="load">intelmpi/2019u4</command>
       </modules>
-      <modules mpilib="impi" compiler="pgi">>
+      <modules mpilib="impi" compiler="pgi">
         <command name="load">intelmpi/2019u3</command>
```

**Comment by rljacob** [comment]
Thanks @jayeshkrishna.  That made me able to build cases.

**Comment by rljacob** [comment]
A 5 day test with --res ne30_oECv3 --compset BGCEXP_CNTL_CNPRDCTC_1850 on compy showed this branch is bfb with the head of maint-1.1.   This is ready for a more thorough review.

**Comment by rljacob** [comment]
These are BFB
BGCEXP_BDRD_CNPRDCTC_20TR
BGCEXP_CNTL_CNPRDCTC_1850

Still to test
BGCEXP_BDRD_CNPECACNT_20TR
BGCEXP_CNTL_CNPECACNT_20TR

**Comment by rljacob** [comment]
BGCEXP_BDRD_CNPECACNT_20TR is also BFB.

I'm going to want to merge this as soon as the last test passes.   If there are any other issues with this PR, let me know soon @kvcalvin @susburrows @jonbob

**Comment by jonbob** [comment]
@rljacob - this PR came up today on the BGC call. What's the status of getting it in?

**Comment by rljacob** [comment]
Still have one more case to test against.  Should finish it this week.

**Comment by rljacob** [comment]
@kvcalvin instead of BGCEXP_CNTL_CNPECACNT_20TR did you mean BGCEXP_CNTL_CNPECACNT_1850?

**Comment by kvcalvin** [comment]
Yes, @rljacob.

**Comment by rljacob** [comment]
BGCEXP_CNTL_CNPECACNT_1850 is BFB. merging.

**Pull request merged**